docs/entities/README.md

```diff
@@ -53,16 +53,18 @@ type CodeChallengeMethod = "S256" | "plain";
 ## Token Entity
 
 The access and refresh token that can be used to authenticate into the resource server.
+The `originatingAuthCodeId` will be present only for tokens originating from the authorization code grant; see the `revokeDescendantsOf()` method on the [Token Repository](https://jasonraimondi.github.io/ts-oauth2-server/repositories/#token-repository) for its significance.
 
 ```typescript
 interface OAuthToken {
   accessToken: string;
   accessTokenExpiresAt: Date;
-  refreshToken?: string;
-  refreshTokenExpiresAt?: Date;
+  refreshToken?: string | null;
```

Comment:
Is there any reason to mark these as nullish? Not 100% opposed or anything, I just want to understand.

Reply:
@jasonraimondi what are your thoughts here, please?